### PR TITLE
{Fabric Version} Add `poisonHealth` Gamerule

### PR DESCRIPTION
**Pull request by FirstMegaGame4**

### CODE REVIEW

File: src/main/resources/not_enough_gamerules.mixins.json [hunk 1]
```diff
@@ -20,7 +22,6 @@
     "entities.VillagerMixin",
     "entities.ZombieMixin",
     "entities.ZombieVillagerMixin",
-    "PistonMixin",
```
Comment (Max094Reikeb) [inline src/main/resources/not_enough_gamerules.mixins.json]: Please sort by alphabetical order

**Review by Max094Reikeb — CHANGES_REQUESTED** [review]

File: src/main/resources/not_enough_gamerules.mixins.json (hunk 1 above)
Comment (FirstMegaGame4) [reply src/main/resources/not_enough_gamerules.mixins.json]: It's listed in alphabetical order
Capitals are sorted separately then lowercase
It's automatic, it does this every time I add a Mixin, it reclassifies correctly

Comment (Max094Reikeb) [reply src/main/resources/not_enough_gamerules.mixins.json]: Well it doesn't sort the way I want it sorted

Comment (FirstMegaGame4) [reply src/main/resources/not_enough_gamerules.mixins.json]: Okay so I'm going to put the weird sorting back :(

**Review by Max094Reikeb — APPROVED** [review]

**Pull request merged**